source/configure/site-configuration-settings.rst

```diff
@@ -252,12 +252,22 @@ Forgot Password custom link
   This configuration setting applies to all Mattermost clients, including web, desktop app, and mobile app. You can control whether the **Forgot Password** link is visible or hidden in clients by going to **Authentication > Password > Enable Forgot Password Link**. See the :ref:`configuration <configure/authentication-configuration-settings:enable forgot password link>` documentation for details.
 
 
+.. config:setting:: report-a-problem-type
+  :displayname: Report a Problem type (Customization)
+  :systemconsole: Site Configuration > Customization
+  :configjson: .SupportSettings.ReportAProblemType
+  :environment: MM_SUPPORTSETTINGS_REPORTAPROBLEMTYPE
+  :description: Select how the ‘Report a Problem’ option behaves. Choosing ‘Custom link’ or ‘Email address’ allows you to provide a URL or address in the next field. ‘Hide link’ removes the ‘Report a Problem’ option from the app.
+
+Report a Problem type
+~~~~~~~~~~~~~~~~~~~~~
+
 .. config:setting:: report-a-problem-link
   :displayname: Report a Problem link (Customization)
   :systemconsole: Site Configuration > Customization
   :configjson: .SupportSettings.ReportAProblemLink
   :environment: MM_SUPPORTSETTINGS_REPORTAPROBLEMLINK
-  :description: This field sets the URL for the **Report a Problem** link in the channel header **Help** menu.  Default value is **https://about.mattermost.com/default-report-a-problem**.
+  :description: Users will be directed to this link when they choose ‘Report a Problem’.
```

Comment:
The default is no longer this one, but depends on whether the user has a license or not.